packages/feedback-service/service.ts

```diff
@@ -1,73 +1,72 @@
-import dotenv from 'dotenv-safe';
-if ( process.env.NODE_ENV === 'test' ) {
-  dotenv.config( { path: '.test.env' } );
-} else {
-  dotenv.config();
-}
-
 import express from 'express';
 import { ApolloServer } from 'apollo-server-express';
 import http from 'http';
-const { ApolloLogExtension } = require( 'apollo-log' );
+const { ApolloLogExtension } = require('apollo-log');
 import mongoose from 'mongoose';
 
 import gqlSchema from './src/typedef.graphql';
 import { FeedbackResolver as resolver } from './src/resolver';
+import cookieParser = require('cookie-parser');
 
 /* Setting port for the server */
 const port = process.env.PORT || 8080;
 
 const app = express();
-const extensions = [ () => new ApolloLogExtension( {
+
+// Mount cookie parser
+app.use(cookieParser());
+
+const extensions = [() => new ApolloLogExtension({
   level: process.env.NODE_ENV === 'test' ? 'silent' : 'info',
   timestamp: true,
   prefix: 'Feedback Service:'
-} ) ];
+})];
```

Comment:
Remove the ApolloLogExtension as the package has been deprecated.

Use the plugins property in the ApolloServer constructor. (Refer the [notifications-service/service.ts#74](https://github.com/1-Platform/one-platform/blob/master/packages/notifications-service/service.ts#L74))